packages/serverless-nextjs-component/__tests__/build.test.js

```diff
@@ -236,6 +236,8 @@ describe("build tests", () => {
       expect(mockCloudFront).toBeCalledWith({
         defaults: {
           allowedHttpMethods: expect.any(Array),
+          queryString: true,
+          cookies: 'all',
```

Comment:
```suggestion
          cookies: "all",
```
The single quotes are breaking the linter :-)